bootstrap/kubeadm/main.go

```diff
@@ -102,7 +102,7 @@ func InitFlags(fs *pflag.FlagSet) {
 		"The amount of time the bootstrap token will be valid")
 
 	fs.IntVar(&webhookPort, "webhook-port", 0,
-		"Webhook Server port, disabled by default. When enabled, the manager will only work as webhook server, no reconcilers are installed.")
+		"Webhook Server port, disabled by default.")
```

Comment:
Should this be updated to enabled by default and match the current webhook default of port `9443`?

Reply:
We are consistent with kubebuilder (webhook disabled by default, but then activated in the manifest via `bootstrap/kubeadm/config/default/manager_webhook_patch.yaml`)
But I can change if there is an agreement.

Reply:
Let's enable it by default

Reply:
Done!